react/src/components/ServiceLauncherModal.tsx

```diff
@@ -432,6 +470,13 @@ const ServiceLauncherModal: React.FC<ServiceLauncherProps> = ({
                 model_definition_path: values.modelDefinitionPath,
               },
             };
+            if (baiClient.supports('modify-endpoint-environ')) {
```

Comment:
There is no `modify-endpoint-environ` in [_updateSupportList](https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2450/refactor-Change-the-items-on-the-summary-page-to-match-BAIBoard)